src/displayapp/screens/WatchFaceTerminal.cpp

```diff
@@ -0,0 +1,252 @@
+#include "WatchFaceTerminal.h"
```

Comment:
please use includes relative to the src directory, helps me greatly with the simulator
```suggestion
#include "displayapp/screens/WatchFaceTerminal.h"
```